src/test/suite/fixtures/recorded/languages/css/chuckArgue5.yml

```diff
@@ -0,0 +1,29 @@
+languageId: css
```

Comment:
I copied these tests over from the SCSS test folder so the names are messed up, nbd? I'm happy to clean it up as well if you feel strongly about it.

Reply:
Yeah that's fine